static/src/javascripts/projects/common/modules/support/banner.ts

```diff
@@ -161,6 +162,8 @@ const buildBannerPayload = async (
 		browserId: (await hasCmpConsentForBrowserId()) ? browserId : undefined,
 		purchaseInfo,
 		isSignedIn: !!isSignedIn,
+		lastOneOffContributionDate:
+			getLastOneOffContributionDate() ?? undefined,
```

Comment:
This is needed because the implementation of `getLastOneOffContributionDate` on frontend can return `null`?

Reply:
yes, and maybe that should just be changed to return `undefined`, though hopefully this code will not be around for long...